library/src/com/google/maps/android/PolyUtil.java

```diff
@@ -284,6 +286,102 @@ private static boolean isOnSegmentGC(double lat1, double lng1, double lat2, doub
     }
 
     /**
+     * Simplifies the given line using the Douglas-Peucker decimation algorithm.  Increasing the
+     * tolerance will result in fewer points in the simplified line.
+     *
+     * @param line input line to be simplified
+     * @param simplifiedLine a simplified line produced by the Douglas-Peucker algorithm
+     * @param tolerance in meters.  Increasing the tolerance will result in fewer points in the
+     *                  simplified line.
+     */
+    public static void simplify(List<LatLng> line, List<LatLng> simplifiedLine, double tolerance) {
```

Comment:
That's a good question - I think so, although I'll run some tests to see for sure.